cg/meta/orders/microbial_fastq_submitter.py

```diff
@@ -0,0 +1,117 @@
+        self._fill_in_sample_ids(samples=status_data["samples"], lims_map=lims_map)
+        new_samples: list[Sample] = self.store_items_in_status(
+            customer_id=status_data["customer"],
+            order=status_data["order"],
+            ordered=project_data["date"] if project_data else datetime.now(),
+            ticket_id=order.ticket,
+            items=status_data["samples"],
+        )
+        self._add_missing_reads(new_samples)
+        return {"project": project_data, "records": new_samples}
+
+    @staticmethod
+    def order_to_status(order: OrderIn) -> dict:
+        """Convert order input for microbial samples."""
+        return {
+            "customer": order.customer,
+            "order": order.name,
+            "samples": [
+                {
+                    "application": sample.application,
+                    "comment": sample.comment,
+                    "data_analysis": sample.data_analysis,
+                    "data_delivery": sample.data_delivery,
+                    "name": sample.name,
+                    "priority": sample.priority,
+                    "volume": sample.volume,
+                    "control": sample.control,
+                }
+                for sample in order.samples
+            ],
+        }
+
+    def store_items_in_status(
+        self,
+        customer_id: str,
+        order: str,
+        ordered: datetime,
+        ticket_id: str,
+        items: list[dict],
+    ) -> list[Sample]:
+        customer: Customer = self._get_customer(customer_id)
+        new_samples: list[Sample] = []
+        for sample in items:
+            case_name: str = sample["name"]
+            case: Case = self._create_case_for_sample(sample, customer, case_name)
+            db_sample: Sample = self._create_sample(sample, order, ordered, ticket_id)
+            db_sample = self._add_application_to_sample(
+                sample=db_sample, application_tag=sample["application"]
+            )
+            case_sample: CaseSample = self.status.relate_sample(
+                case=case, sample=db_sample, status=StatusEnum.unknown
+            )
+            self.status.add_multiple_items_to_store([case, db_sample, case_sample])
+            new_samples.append(db_sample)
+        self.status.commit_to_store()
+        return new_samples
+
+    def _get_customer(self, customer_id: str) -> Customer:
+        if customer := self.status.get_customer_by_internal_id(customer_id):
+            return customer
+        raise EntryNotFoundError(f"could not find customer: {customer_id}")
+
+    def _create_case_for_sample(self, sample: dict, customer: Customer, case_name: str) -> Case:
+        if case := self.status.get_case_by_name_and_customer(
+            case_name=case_name, customer=customer
+        ):
+            return case
+        case: Case = self.status.add_case(
+            data_analysis=Workflow.FASTQ,
+            data_delivery=DataDelivery.FASTQ,
```

Comment:
Data delivery should be picked from the order. There is also a "no delivery" option.